packages/vue-app/template/utils.js

```diff
@@ -134,6 +134,7 @@ export async function setContext(app, context) {
     // Only set once
     if (context.req) app.context.req = context.req
     if (context.res) app.context.res = context.res
+    if (context.ssrContext) app.context.ssrContext  = context.ssrContext
```

Comment:
We have a bunch of inline if's here! :see_no_evil:

Reply:
Yeah. These were from before. PS i think sometimes one line ifs are more readable :see_no_evil:

Reply:
@pi0 They sometimes are *but* can lead to errors quite frequently as well :see_no_evil: 

We could use lodash's `pick` +`Object.assign` in this specific case, couldn't we? :thinking: 

`Object.assign(context, objectWithPickedResReqAndSsrContextHere)`

Reply:
Using lodash has both bundling and perf overhead. We can use a for loop :)

Reply:
That's fine as well :+1:

Reply:
![image](https://user-images.githubusercontent.com/5158436/52376745-23687080-2a78-11e9-9da9-507d3e96d8dd.png)

@manniL I proposed this. But still, think it just introduces more complexity and perf overhead. :thinking:

Reply:
Yeah, you are right. Let's stick with the `if`s. Would you leave them inlined or use "full" blocks?

Reply:
Will do full blocks to comply with lint rules. We can talk more about it later :)